File: RetroFit/app/src/main/java/fr/coopuniverse/api/retrofit/Reponse.java
package fr.coopuniverse.api.retrofit;

import org.json.JSONObject;

public class Reponse {
    private int exitCode;
    private Data data;


    public int getExitCode() {
        return exitCode;
    }

    public void setExitCode(int exitCode) {
        this.exitCode = exitCode;
    }

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }
    @Override
    public String toString()
    {
        if(data != null)
            return "Exit Code : "+ exitCode + " Data : " + data.toString();
        else
            return "Exit Code : "+ exitCode;
    }
}

File: client_kotlin/app/src/main/java/fr/coopuniverse/api/pokeapi/activity/httpRequestManager/Reponse.kt
package fr.coopuniverse.api.pokeapi.activity.httpRequestManager


class Reponse(var id: String, var connect: Boolean = false, var exitCode: Int = -1, var data: Data = Data(-1)) {


    override fun toString(): String {
        return if (data != null)
            "Connect : " + connect + " Exit Code : " + exitCode + " Data : " + data!!.toString()
        else
            "Exit Code : $exitCode"
    }
}
